### DISCUSSION TITLE
Not an issue, but is there analogous software for Download?

### DISCUSSION THREAD

**Discussion by raimondsre** [labbots/google-drive-upload#180] [Q&A]
I have tested ~10 google drive related programms and Gupload is the single that performed the task - upload to institutions shared drive with limited permissions. 

May i ask if you could recommend software for Download? Unfortunately i have tried every single popular project but they just do not work in this case.

**Comment by Akianonymus** [comment]
https://github.com/Akianonymus/gdrive-downloader
